src/tiled/propertybrowser.cpp

```diff
@@ -946,6 +961,10 @@ void PropertyBrowser::updateProperties()
         const Tileset *tileset = static_cast<const Tileset*>(mObject);
         mIdToProperty[NameProperty]->setValue(tileset->name());
         mIdToProperty[TileOffsetProperty]->setValue(tileset->tileOffset());
+        mIdToProperty[SourceImageProperty]->setValue(tileset->imageSource());
+        mIdToProperty[TileSizeProperty]->setValue(tileset->tileSize());
+        mIdToProperty[MarginProperty]->setValue(tileset->margin());
+        mIdToProperty[SpacingProperty]->setValue(tileset->tileSpacing());
```

Comment:
You'll run into a crash if you conditionally create these properties but unconditionally set their value. Please do test your change. :-)